### DISCUSSION TITLE
Pretix Scan - Setup from mobile email

### DISCUSSION THREAD

**Discussion by jeverington** [pretix/pretix#4214] [Q&A]
Many volunteers who we get setup with Pretix Scan will get their emails on a mobile device. Is it possible to be able to somehow send a link (deeplink?) that can configure Pretix Scan instead of having the QRCode? Currently we have to print the codes out for them to scan again on their phones.

**Comment by pc-coholic** [comment]
That is something that we're looking into, but there is no commitment/fixed timeline for this feature, sorry.

**Reply by jeverington** [reply]
Thanks Martin!
